### PR TITLE
[REF] Use expected errors of pylint based on dictionaries instead of a global total value

### PR DESCRIPTION
**Pull request by hugho-ad**

### CODE REVIEW

**Comment by moylop260** [comment]
Could you check the red result of CI?

**Comment by moylop260** [comment]
FYI the CI red is unrelated fixed from https://github.com/OCA/maintainer-quality-tools/pull/436

**Comment by moylop260** [comment]
cc @dreispt @pedrobaeza @lasley @alan196 @jjscarafia

**Comment by dreispt** [comment]
I prefer to wait for #436 and a rebase.

**Comment by hugho-ad** [comment]
@dreispt can you help me, I don't know why the coverage builds does not includes the job that tests the lints, Do you have  any clue?
There is missing the job with the env vars `LINT_CHECK="1" TESTS="0"`

image

**Comment by nhomar** [comment]
This PR is fundamental for a clean cleanup into #500 I will wait for this merge to continue that work the current version of the scripts do not need to be refactored twice due to that is partially done here.

**Comment by moylop260** [comment]
codecov don't find coverage report but exists because coveralls is using it

I have created a question from gitter...
https://gitter.im/codecov/support?at=59f90a3eb20c64242970cd7b

But this is a issue unrelated.

What do you think about merge it, because is a long time PR waiting for merge?
@dreispt @pedrobaeza @lasley

**Comment by moylop260** [comment]
FYI I found the codecov issue is because is not supported a custom named of the coverage result file.
So I just renamed it using the standard one and seems is working.

Please, review it

**Comment by hugho-ad** [comment]
@moylop260 awesome work!

Can you point us the cidecov issue ?

Regards!

**Comment by moylop260** [comment]
Yes of course
I updated the comment https://github.com/OCA/maintainer-quality-tools/pull/434#issuecomment-340939234

**Comment by moylop260** [comment]
Dummy PR https://github.com/OCA/stock-logistics-warehouse/pull/350

**Comment by moylop260** [comment]
This PR is ready to review it

File: travis/travis_install_nightly [hunk 1]
```diff
@@ -3,7 +3,7 @@
 set -e
 
 if [ "${LINT_CHECK}" != "0" ]; then
-    pip install -q flake8=="3.4.1" Click pylint-mccabe
+    pip install -Uq flake8=="3.4.1" Click pylint-mccabe pbr
```
Comment (lasley) [inline travis/travis_install_nightly]: pbr?

Comment (moylop260) [reply travis/travis_install_nightly]: pylint-odoo is using pbr and I had problems locally with the order of installation like as the last one with saas npm package

**Pull request merged**